measuring-client/src/main/java/de/cyface/app/ui/notification/CameraEventHandler.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 Cyface GmbH
+ * Copyright 2017-2022 Cyface GmbH
  *
  * This file is part of the Cyface App for Android.
  *
```

Comment:
I just added migration steps which I applied to `main` branch when I upgraded `main` to `targetVersion 31` which we do in this branch now, too.